site2/docs/client-libraries-cpp.md

```diff
@@ -4,298 +4,173 @@ title: Pulsar C++ client
 sidebar_label: "C++"
 ---
 
-You can use Pulsar C++ client to create Pulsar producers and consumers in C++.
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
 
-All the methods in producer, consumer, and reader of a C++ client are thread-safe.
+You can use a Pulsar C++ client to create producers, consumers, and readers.
 
-## Supported platforms
+All the methods in producer, consumer, and reader of a C++ client are thread-safe. You can read the Doxygen-generated [API docs](/api/cpp) for the C++ client
 
-Pulsar C++ client is supported on **Linux**, **macOS** and **Windows** platforms.
+## Installation
 
-Doxygen-generated API docs for the C++ client are available [here](/api/cpp).
+Use one of the following methods to install a Pulsar C++ client.
 
-## Linux
+### Brew
 
-:::note
-
-You can choose one of the following installation methods based on your needs: Compilation, Install RPM or Install Debian.
+Install the latest tagged version with the library and headers:
 
-:::
-
-### Compilation
+```bash
+brew install libpulsar
+```
 
-#### System requirements
+For more information, read [libpulsar formula's homepage](https://formulae.brew.sh/formula/libpulsar).
 
-You need to install the following components before using the C++ client:
+### Deb
 
-* [CMake](https://cmake.org/)
-* [Boost](http://www.boost.org/)
-* [Protocol Buffers](https://developers.google.com/protocol-buffers/) >= 3
-* [libcurl](https://curl.se/libcurl/)
-* [Google Test](https://github.com/google/googletest)
+1. Download any one of the Deb packages:
 
-1. Clone the Pulsar repository.
+<Tabs>
+<TabItem value="client">
 
-```shell
-git clone https://github.com/apache/pulsar
+```bash
+wget @pulsar:deb:client@
```

Comment:
Could you explain what does `@pulsar:deb:client@` mean? I see it doesn't change even in the preview page.

Reply:
It will be replaced with link like `https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-2.10.1/DEB/apache-pulsar-client.deb` according to the real version - since our document are versioned.

The actual resolving and replacing logic is somewhere within https://github.com/apache/pulsar-site. I suggest @urfreespace or @tuteng can give some inputs here. I'm confident that this URL is the correct one.

Reply:
I think we'd better add it to the guide for document contributors.